pkg/webhooks/admission/jobs/mutate/mutate_job.go

```diff
@@ -204,3 +208,26 @@ func mutateSpec(tasks []v1alpha1.TaskSpec, basePath string) *patchOperation {
 		Value: tasks,
 	}
 }
+
+func patchDefaultPlugins(job *v1alpha1.Job) *patchOperation {
+	if job.Spec.Plugins == nil {
+		return nil
+	}
+	plugins := map[string][]string{}
+	for k, v := range job.Spec.Plugins {
+		plugins[k] = v
+	}
+	if _, ok := job.Spec.Plugins["tensorflow"]; ok {
+		if _, ok := plugins["svc"]; !ok {
+			plugins["svc"] = []string{}
+		}
+		if _, ok := plugins["env"]; !ok {
+			plugins["env"] = []string{}
+		}
+	}
+	return &patchOperation{
+		Op:    "replace",
+		Path:  "/spec/plugins",
+		Value: plugins,
+	}
+}
```

Comment:
It's better to add some comments or log here to clarify that tensorflow plugin depends on svc plugin )